dockerfiles/haskell/3.0.0.0/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:22.04
+
+# Install necessary packages for GHC (compiler)
+RUN apt-get update \
+    && apt-get -y --no-install-recommends install \
+    build-essential \
+    curl \
+    libgmp-dev \
+    ghc cabal-install \
+    && rm -rf /var/lib/apt/lists/*
+    
+# Maybe add Stack (another build tool for Haskell), up to Instructor?
+# RUN curl -sSL https://get.haskellstack.org/ | sh
```

Comment:
I'm not too familiar with Haskell build systems. Would stack and cabal be mutually exclusive for a given assignment?

If so, we could make two tags. One could be `3-cabal` and the other `3-stack`.

Reply:
I don't think so, they're used together

Reply:
You may not need it but I'm wondering if it's something helpful to include. Based on what I'm seeing online, it appears stack and cabal are their own separate build systems so you wouldn't mix the two.